pkg/resource/nodecadaemon.go

```diff
@@ -38,7 +38,7 @@ spec:
       tolerations:
       - effect: NoSchedule
         key: node-role.kubernetes.io/master
-        operator: Exists
+        operator: Exist
```

Comment:
?

all the others in this PR are Exists, why is this changing to Exist?

Reply:
It's a mistake on my end. Thanks for catching it. It should be `Exists`.